api-docs/v2/api-operations/methods/GET_listRateLimits_v2_account_id.rst

```diff
@@ -0,0 +1,183 @@
+| 404     | Not Found             | The requested item was not found.           |
++---------+-----------------------+---------------------------------------------+
+| 405     | Method Not Allowed    | The method specified in the request is      |
+|         |                       | not allowed for the resource identified by  |
+|         |                       | the request URI.                            |
++---------+-----------------------+---------------------------------------------+
+| 413     | Over Limit            | The request exceeds the rate limit or quota.|
++---------+-----------------------+---------------------------------------------+
+| 415     | Unsupported Media     | The server won't service the                |
+|         | Type                  | request because the entity of the request   |
+|         |                       | is in a format not supported by the         |
+|         |                       | requested resource for the requested        |
+|         |                       | method.                                     |
++---------+-----------------------+---------------------------------------------+
+| 503     | Service Unavailable   | The service is not available.               |
++---------+-----------------------+---------------------------------------------+
+
+Request
+""""""""""""""""
+
+This table shows the URI parameters for the request:
+
++-----------------------+---------+---------------------------------------------+
+| Name                  | Type    | Description                                 |
++=======================+=========+=============================================+
+| ``{TENANT_ID}``       | ​String | The account ID of the account owner.        |
++-----------------------+---------+---------------------------------------------+
+
+**Example:  List rate limits, request**
+
+.. code::  
+
+    GET /v2/123456/ratelimits HTTP/1.1
+    Host: global.dns.rackspacecloud.com
+    Accept: application/json
+    Content-Type: application/json
+
+This operation does not accept a request body.
+
+Response
+""""""""""""""""
+
+This table shows the body parameters for the response:
+
++--------------------------------+----------------------+----------------------+
+|Name                            |Type                  |Description           |
++================================+======================+======================+
+|**limits**                      |Object                |A container of limits.|
++--------------------------------+----------------------+----------------------+
+|limits.\ **rate**               |Array                 |An array of rates.    |
+|                                |                      |limits objects.       |
++--------------------------------+----------------------+----------------------+
+|limits.rate. \ **limit**        |Array                 |An array of limits for|
+|                                |                      |a rate.               |
```

Comment:
I'd say `array of limits for a URI` here. Because the limits that are present apply to a specific regex pattern, and in our case, those are always matching one URI.